maps/src/G3SkyMapMask.cxx

```diff
@@ -639,9 +639,16 @@ PYBINDINGS("maps")
 	    "Apply a mask in-place to the mask, optionally inverting which "
 	    "pixels are zeroed.  If inverse = False, this is equivalent to "
 	    "in-place element-wise logical-and with the mask.")
+#ifdef __clang__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wself-assign-overloaded"
+#endif
```

Comment:
I would lean in favor of globally disabling this warning, which seems dubious and will be guaranteed to trigger on certain idioms in Boost.

Reply:
Fair.  Does it make sense to add to the pybindings.h header as a global pragma for any source files that use that header?

Reply:
I've done the above, rather than opening the door again to creating a laundry list of particular warnings to be disabled in the cmake file.

Reply:
I'd honestly prefer it in CMakeLists.txt to a semi-hidden global #pragma

Reply:
(After conversation with Sasha offline, this seems fine)